### PR TITLE
wip: universal entry point

### PR DESCRIPTION
**Pull request by generall**

### CODE REVIEW

File: tests/test_qdrant_client.py [hunk 1]
```diff
@@ -62,44 +64,66 @@ def create_random_vectors():
     return vectors_path
 
 
-def test_rest_init():
+def test_client_init():
+    import tempfile
```
Comment (joein) [inline tests/test_qdrant_client.py:68]: why is it here

**Review by joein — APPROVED** [review]
Well, yeah, considered a similar option too

Looks ok, don't really like the naming though)) this `impl` thing and the fact that the attr is not protected

File: tests/test_qdrant_client.py (hunk 1 above)
Comment (generall) [reply tests/test_qdrant_client.py:68]: to test `path=/tmp/...`

Comment (joein) [reply tests/test_qdrant_client.py:68]: I mean why the import is inside the function, any particular reason?

Comment (joein) [reply tests/test_qdrant_client.py:68]: it's just a nit comment

**Pull request merged**